pkg/common/peertracker/npipe_windows.go

```diff
@@ -44,8 +44,10 @@ func getCallerInfoFromNamedPipeConn(conn net.Conn) (CallerInfo, error) {
 	}, nil
 }
 
+// getNamedPipeClientProcessID retrieves the client process identifier
+// for the specified handle representing a named pipe.
 func getNamedPipeClientProcessID(pipe windows.Handle, clientProcessID *int32) (err error) {
-	r1, _, e1 := syscall.Syscall(procGetNamedPipeClientProcessID.Addr(), 2, uintptr(pipe), uintptr(unsafe.Pointer(clientProcessID)), 0)
+	r1, _, e1 := syscall.SyscallN(procGetNamedPipeClientProcessID.Addr(), 2, uintptr(pipe), uintptr(unsafe.Pointer(clientProcessID)), 0)
```

Comment:
```suggestion
	r1, _, e1 := syscall.SyscallN(procGetNamedPipeClientProcessID.Addr(), uintptr(pipe), uintptr(unsafe.Pointer(clientProcessID)))
```

Reply:
This use of SyscallN is incorrect. SyscallN just takes a slice of args and determines the correct number of args using that.